RELEASE_NOTES.md

```diff
@@ -1,5 +1,56 @@
 # Release Notes
 
+# Staging 2019/10/09
+
+## Ingest
+### Core (dfdcbe0)
+* Lazy load DB Refs for performance improvements
+* Validator (9ddaeaf)
+* Fixes for retries
+* Return Validation Report
+### UI (2e1cd40)
+* Security Fixes
+### Client (2787fe7)
+* Fix to return all files in get_bundle
+* Fix importer code
+* Use new SchemaTemplate SchemaParser
+* Broker (810771c)
+* Exporter (d75dee7)
+* Staging Manager (47cbd64)
+
+## Secondary Analysis
+### Lira: v0.22.3
+* Write the timestamp to inputs for adapters
+* Update the SmartSeq2 query to match mouse data and remove the unnecessary "should" clause
+* Update testing data to reflect moving adapter pipelines into a new repo
+* Update scripts that get/create/delete DSS subscriptions to use elasticsearch as the default query type
+* Update pipeline-tools version to fix getting metadata from the production data store
+* Add config flag for testing mode
+* Add hash-id backfill script
+
+### Adapter-pipelines [formerly part of pipeline-tools]: v1.1.0
+* Add the ability to disable call-caching in the prep task using a timestamp parameter
+* Remove file format input parameter (moved into the pipeline-tools code)
+* Migrate the following existing adapter pipelines and associated inputs from the pipeline-tools repo:
+** optimus
```

Comment:
I think these bullet points are accidentally doubled.

Reply:
not being a markdown expert - had to figure out how to increase indentation for the list.  figured it out now.